x-pack/plugins/security_solution/public/common/containers/sourcerer/get_sourcerer_data_view.ts

```diff
@@ -10,9 +10,10 @@ import { ensurePatternFormat } from '../../../../common/utils/sourcerer';
 
 export const getSourcererDataView = async (
   dataViewId: string,
-  dataViewsService: DataViewsContract
+  dataViewsService: DataViewsContract,
+  refreshFields = false
 ) => {
-  const dataViewData = await dataViewsService.get(dataViewId);
+  const dataViewData = await dataViewsService.get(dataViewId, true, refreshFields);
```

Comment:
this `true` value is the default value for the positional argument of `displayErrors=true`.